Categories changes (#379)
tachiyomiorg/website: src/help/guides/categories.md

# Categories

**Categories** help you store your bookmarked manga in custom tabs, which keeps your library neat and organized. You can name and sort them however you want according to your preference (ie. _Genre_, _Reading Status_, etc).

> You can even add your bookmarked manga in multiple categories and can easily control the update options through Library settings and automatically download new manga chapters from certain categories (if **Download new chapters** option is enabled in **Downloads** settings).

Reviewer: Needs a line-break here so it doesn't look too clumped up.
```suggestion

You can even add your bookmarked manga in multiple categories and can easily control the update options through Library settings and automatically download new manga chapters from certain categories (if **Download new chapters** option is enabled in **Downloads** settings).
```

## Create a category

> To create a category follow the steps below.

Reviewer: Should this use a comma? It feels like it requires a short-pause

```suggestion
To create a category, follow the steps below.
```

::: guide
1. Go to <Navigation item="more"/> → <Navigation item="categories"/>.

> 1. Press on **+ Add**  button on bottom right.

Reviewer: Would this make more sense?
```suggestion
1. Press the **+ Add**  button in the bottom right.
```
